addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -161,6 +161,15 @@ var MassMailingFieldHtml = FieldHtml.extend({
         return this._super.apply(this, arguments);
     },
 
+    /**
+     * @override
+     */
+    _renderReadonly: function () {
+        if (!this.value) {
```

Comment:
How come `this.value` is falsy? Is that expected in this case?

Reply:
yes it's expected, this is same as above **_renderEdit** function, when value not found it will be assigning a value to the **value** property of the current object.
 
Thank you..

Reply:
Ah right, that makes sense. Looks good to me :+1:

Reply:
Hello @tde-banana-odoo
I kindly request that you inform me if any further changes are necessary.

Thankyou...